pkg/vm/engine/tae/txn/txnimpl/sysblock.go

```diff
@@ -252,9 +252,10 @@ func (blk *txnSysBlock) getRelTableData(colIdx int) (view *model.ColumnView, err
 		case catalog.SystemRelAttr_Persistence:
 			colData.Append([]byte(catalog.SystemPersistRel))
 		case catalog.SystemRelAttr_Kind:
-			colData.Append([]byte(catalog.SystemOrdinaryRel))
+			colData.Append([]byte(table.GetSchema().Relkind))
 		case catalog.SystemRelAttr_CreateSQL:
-			colData.Append([]byte("todosql"))
+			//colData.Append([]byte("todosql"))
```

Comment:
I think CreateSQL use for  CREATE STATEMENT.  not for load statement。
you can confirm with pengzhen